tests/karate/src/test/java/fe/utils/verification/MessageVerification.java

```diff
@@ -30,7 +30,8 @@ public boolean verifyMessageIdField(String message) {
     String signature = messageFieldJson.get(SIGNATURE);
     String msgId = messageFieldJson.get(MESSAGE_ID);
     try {
-      return msgId.equals(JsonConverter.hash(data.getBytes(), signature.getBytes()));
+      JsonConverter jsonConverter = new JsonConverter();
+      return msgId.equals(jsonConverter.hash(data.getBytes(), signature.getBytes()));
```

Comment:
What is the reason of this change ?

Reply:
The usage of the method hash always required an instance of `JsonConverter`. But @Warhedgehog changed the hash method to static, which made all the other usages where hash was invoked on a` JsonConverter` object illegal and there was quite a few of them. So I had to switch back to the non static usage of the hash method and update all the places where it was used which was only 3 times so changes are not drastic and the functionality still remains the same.

Reply:
Ok